### DISCUSSION TITLE
Why "assign AGIC identity to have operator access over AppGw identity" is needed ?

### DISCUSSION THREAD

**Discussion by yukit7s** [Azure/application-gateway-kubernetes-ingress#1121] [Q&A] [answered]
see https://github.com/Azure/application-gateway-kubernetes-ingress/blob/master/docs/features/appgw-ssl-certificate.md#configure-certificate-from-key-vault-to-appgw

Why "assign AGIC identity to have operator access over AppGw identity" is needed ?
Even if I have skipped this step, I could use AGIC with SSL certification.

**Comment by akshaysngupta** [comment, marked as answer]
since AGIC is performing a PUT operation on Application Gateway and the payload will have a reference to User assigned identity, it is needed that AGIC has appropriate permissions over the identity. The minimum permission needed is `Managed Identity Operator` access. Without this access, this operation will be forbidden.
